tests/e2e/test_small_file_workload.py

```diff
@@ -43,23 +40,32 @@ class TestSmallFileWorkload(E2ETest):
     """
 
     @pytest.mark.parametrize(
-        argnames=["file_size", "files", "threads", "samples"],
+        argnames=["file_size", "files", "threads", "samples", "interface"],
         argvalues=[
-            pytest.param(*[4, 50000, 4, 1], ),
-            pytest.param(*[16, 50000, 4, 1], ),
-            pytest.param(*[16, 200000, 4, 1], ),
+            pytest.param(*[4, 50000, 4, 1, constants.CEPHBLOCKPOOL], ),
+            pytest.param(*[16, 50000, 4, 1, constants.CEPHBLOCKPOOL, ),
+            pytest.param(*[16, 200000, 4, 1, constants.CEPHBLOCKPOOL], ),
+            pytest.param(*[4, 50000, 4, 1, constants.CEPHFILESYSTEM], ),
+            pytest.param(*[16, 50000, 4, 1, constants.CEPHFILESYSTEM], ),
+            pytest.param(*[16, 200000, 4, 1, constants.CEPHFILESYSTEM], ),
         ]
     )
     @pytest.mark.polarion_id("OCS-1295")
-    def test_smallfile_workload(self, ripsaw, file_size, files, threads, samples):
+    def test_smallfile_workload(self, ripsaw, file_size, files, threads, samples, interface):
         """
         Run SmallFile Workload
         """
         log.info("Apply Operator CRD")
         ripsaw.apply_crd('resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml')
-
-        log.info("Running SmallFile bench")
         sf_data = templating.load_yaml(constants.SMALLFILE_BENCHMARK_YAML)
+        if interface == constants.CEPHBLOCKPOOL:
```

Comment:
Replacing if else:
```
storageclass = constants.DEFAULT_STORAGECLASS_RBD if interface == constants.CEPHBLOCKPOOL else constants.DEFAULT_STORAGECLASS_CEPHFS
log.info(f"Using {storageclass} Storageclass")
```

Reply:
The line is 140 and flake8 complains

Reply:
line too long (140 > 120 characters)cornflakes(E501)

Reply:
I moved the log.info() out, but kept the if/else as is to limit < 80 chars to stop linter from complaining. I will resolve the comment for now.